frontend/packages/dev-console/src/plugin.tsx

```diff
@@ -532,6 +534,32 @@ const plugin: Plugin<ConsumedExtensions> = [
         ).default,
     },
   },
+  {
+    type: 'Page/Route',
+    properties: {
+      exact: true,
+      path: ['/samples/all-namespaces', '/samples/ns/:ns'],
+      loader: async () =>
+        (
+          await import(
+            './components/import/SamplesCatalog' /* webpackChunkName: "dev-console-samples-catalog" */
+          )
+        ).default,
+    },
+  },
+  {
+    type: 'Page/Route',
+    properties: {
+      exact: true,
+      path: ['/import-sample/all-namespaces', '/import-sample/ns/:ns'],
+      loader: async () =>
+        (
+          await import(
+            './components/import/ImportSamplePage' /* webpackChunkName: "dev-console-import-sample" */
+          )
+        ).default,
```

Comment:
`'/import-sample/all-namespaces'` this route results in a white screen with error `Cannot read property 'tags' of undefined`
and so will this `'/import-sample/ns/:ns'`. I guess we should redirect the user to `'samples/ns/:ns'` if there is no query params present in the URL.

Reply:
Since `/import-sample` is highly dependent on the query params, can we remove it and use `/samples/ns/:ns` with query params for the `ImportSamplePage` component. Not sure about this. WDYT?

Reply:
Fixed.